### PR TITLE
scripts: Add a script to generate incremental OTA from target zips

### PR DESCRIPTION
**Pull request by Dhruvgera**
Removes the need for installing full package, and saves data bandwidth

### CODE REVIEW

**Comment by Dhruvgera** [comment]
This is for building incremental zips for AOSP

File: android-incremental-ota.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import os
+print("Incrementals are made using target zips which are found under out/target/product/device/obj/PACKAGING/target_files_intermediates")
```
Comment (kaustubhgupta) [inline android-incremental-ota.py:2]: restyle the whole program with spaces at appropriate places. Refer this link https://www.python.org/dev/peps/pep-0008/

File: android-incremental-ota.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+print("Generating incremental package")
+try:
+		filecheck=open("generator.sh")
+		filecheck.close()
+except:
+    code="./build/make/tools/releasetools/ota_from_target_files  --block -i old new incremental.zip"
+    saveFile2=open("generator.sh",'w')
+    saveFile2.write(str(code))
+    saveFile2.close()
+fin = open("generator.sh", "rt")
```
Comment (kaustubhgupta) [inline android-incremental-ota.py:14]: use better variable names

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Please make the changes

**Comment by kaustubhgupta** [comment]
@Dhruvgera are you working on this?

**Comment by Dhruvgera** [comment]
> @Dhruvgera are you working on this?

Yes, would be making changes

**Comment by kaustubhgupta** [comment]
Please make another PR as this violated contribution guidelines:

**Comment by Dhruvgera** [comment]
Can I know which guidelines were violated? I wasn't able to make the proper changes due to time constraints

**Pull request closed without merging**